go.mod

```diff
@@ -17,6 +17,11 @@ require (
 	github.com/Azure/go-autorest/autorest/azure/auth v0.5.13
 	github.com/BurntSushi/toml v1.4.0
 	github.com/aws/aws-sdk-go v1.55.2
+	github.com/aws/aws-sdk-go-v2 v1.26.0
```

Comment:
(super-nitpick) I wonder how hard it woud be to split the commit that just add dependencies to vendor/* and the one that actually chnage the code. I like to skip commits that are essentially um-reviewable (like adding the new aws sdk with hundrets of thousands of lines) and by splitting between imported code and changed code this would be slightly eaiser. But I understand the argument to combine them (plus it's a bit of extra work) so fine to ignore my comment :) 

P.S. What I would *actually* want is to not include depednecies like this in our git tree at all but that is a bigger argument :)

Reply:
No that's fair, I'll rebase